### PR TITLE
Fix import error too general other issues

### PR DESCRIPTION
**Pull request by qiaozha**
fix issues 
#762 
#764 
#763 partly still need cli team to provide a mapping from RP name to current SDK mode  track1 or track2

### CODE REVIEW

File: src/entry.ts [hunk 1]
```diff
@@ -134,6 +129,17 @@ function processGenerationOption(session: Session<CodeModel>) {
         ? AzConfiguration.getValue(CodeGenConstants.compatibleLevel)
         : compatibleLevel;
     const isTrack1 = compatibleLevel === CompatibleLevel.Track1;
+    session.message({
+        Channel: Channel.Warning,
```
Comment (lirenhe) [inline src/entry.ts:133]: Is it better to use information here?

File: readme.az.common.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # configuration for az common
  
 ``` yaml $(az)
```
Comment (lirenhe) [inline readme.az.common.md:3]: suggest updating the document for the default module mode.

Comment (qiaozha) [reply readme.az.common.md:3]: okay.

File: src/entry.ts (hunk 1 above)
Comment (qiaozha) [reply src/entry.ts:133]: I am wondering whether infomation will show up in non-debug mode. Will check that, and change it back if information can show up.

File: src/generate/CodeModelAzImpl.ts [hunk 3]
```diff
@@ -1009,6 +1020,9 @@ export class CodeModelCliImpl implements CodeModelAz {
 
     public get CommandGroup_Mode(): string {
         if (isNullOrUndefined(this.CommandGroup?.language?.['cli']?.extensionMode)) {
+            if (this.IsCliCore && this.Extension_Mode === ExtensionMode.Stable) {
```
Comment (lirenhe) [inline src/generate/CodeModelAzImpl.ts:1023]: why is the logic here?

File: src/entry.ts (hunk 1 above)
Comment (qiaozha) [reply src/entry.ts]: Updated

File: readme.az.common.md (hunk 2 above)
Comment (qiaozha) [reply readme.az.common.md:3]: updated in here https://github.com/Azure/autorest.az/pull/797/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R79-R80

File: src/generate/CodeModelAzImpl.ts (hunk 3 above)
Comment (qiaozha) [reply src/generate/CodeModelAzImpl.ts:1023]: generally, the command group layer mode will take the global extension mode value if there's no command group layer mode specified. 
For example if the global extension mode is set as experimental we don't need to set each command group as experimental again. That's why we have the previous changes here https://github.com/Azure/autorest.az/pull/797/files#diff-2c3dfede02f6d90578baaa72578f79e8572b6cb71c5449f3237b5adfe2f943c4L63-L162 
but issue #763 says that we shouldn't label global extension mode as experimental when generating main repo modules. instead we should put the experimental label down to each generated command group layer.

**Review by lirenhe — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]
LGTM for the fix of #762

**Pull request merged**